tests/sentry/incidents/test_logic.py

```diff
@@ -397,6 +405,30 @@ def test_simple(self):
 
 class GetIncidentSuspectsTest(TestCase, BaseIncidentsTest):
 
+    def test_simple(self):
+        release = self.create_release(project=self.project, version='v12')
+
+        self.repo = Repository.objects.create(
+            organization_id=self.organization.id,
+            name=self.organization.id,
+        )
+        commit_id = 'a' * 40
+        release.set_commits([
+            {
+                'id': commit_id,
+                'repository': self.repo.name,
+                'author_email': 'bob@example.com',
+                'author_name': 'Bob',
+            },
+        ])
+        incident = self.create_incident(self.organization)
+        commit = Commit.objects.get(releasecommit__release=release)
+        IncidentSuspectCommit.objects.create(incident=incident, commit=commit, order=1)
+        assert [commit] == list(get_incident_suspects(incident, [self.project]))
+        assert [] == list(get_incident_suspects(incident, []))
```

Comment:
@markstory Related to the excluded_group, I test that the permissions work correctly here instead by just not including the relevant project.